pkg/oci/instances.go

```diff
@@ -96,7 +96,7 @@ func (cp *CloudProvider) NodeAddresses(ctx context.Context, name types.NodeName)
 // nodeaddresses are being queried. i.e. local metadata services cannot be used
 // in this method to obtain nodeaddresses.
 func (cp *CloudProvider) NodeAddressesByProviderID(ctx context.Context, providerID string) ([]api.NodeAddress, error) {
-	glog.V(4).Infof("NodeAddressesByProviderID(%q) called", providerID)
+	cp.logger.Debugf("NodeAddressesByProviderID(%q) called", providerID)
```

Comment:
These types of log messages are largely useless imo. It'd be more useful to say

`cp.logger.With("providerID", providerID).Debug("fetching node addresses by provider id")`